frontend/app/pods/admin/teams/team/edit/route.js

```diff
@@ -0,0 +1,23 @@
+import Ember from 'ember';
+import TeamRouteMixin from 'rafee/mixins/team-route';
+
+export default Ember.Route.extend(TeamRouteMixin, {
+
+  actions: {
+    //TODO: put this in a mixin
```

Comment:
Can you do the TODO comment? this will be also used by other routes so it should be extracted in a mixin